app/jobs/send_email_to_leaders_job.rb

```diff
@@ -0,0 +1,11 @@
+class SendEmailToLeadersJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(user)
+    User.near([user.latitude, user.longitude],50)
```

Comment:
We renamed this to community_leaders_nearby(latitude, longitude, radius) since we had to change the method signature to pass in latitude, longitude.

There was an alternate way to do this I noticed after re-reading the doc (https://github.com/alexreisner/geocoder). We could have made this an instance method and leveraged Geocoder's .nearbys to avoid passing the lat and long, so it could be something like

```
def community_leaders_within(radius)
  nearbys(radius).tagged_with(User::LEADER)
end
```
I kind of like the latter using nearbys. The class method lets you pass in arbitrary coordinates, but we don't have a use case for that yet.